packages/eslint-config-qlean/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-qlean",
+  "version": "1.0.0",
+  "description": "JavaScript Qlean Style - EsLint Shareable Config",
```

Comment:
ESLint is obvs for JS. And what exactly "shareable" means here?

Reply:
@alextewpin [Shareable configs](https://eslint.org/docs/developer-guide/shareable-configs)